features/ticketing/api/pom.xml

```diff
@@ -8,6 +8,21 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.opennms.features.ticketing</groupId>
   <artifactId>org.opennms.features.ticketing.api</artifactId>
-  <name>OpenNMS Features Ticketing API</name>
-  <url>http://www.opennms.org/</url>
+  <packaging>bundle</packaging>
+  <name>OpenNMS :: Features :: Ticketing :: API</name>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
```

Comment:
I agree. We should call it osgiExecutionEnvironment just to make its usage explicit. Then we can change it later on to Java-1.9 and the variable name doesn't change.